tests/TestCase/PaginationResultTest.php

```diff
@@ -15,6 +16,21 @@
 
 class PaginationResultTest extends TestCase
 {
+    public function setUp(): void
+    {
```

Comment:
According to [Test Case Lifecycle Callbacks](https://book.cakephp.org/4/en/development/testing.html#test-case-lifecycle-callbacks), it is important to call `parent::setUp() when overriding this method.
```suggestion
    {
        parent::setUp();

```